src/GeositeFramework/plugins/layer_selector_v2/layers.json

```diff
@@ -150,5 +150,43 @@
                 "downloadUrl": "http://www.tceq.texas.gov/assets/public/gis/exports/au_oysterwaters.zip"
             }
         ]
+    },
+    {
+        "displayName": "WMS Example Layers",
+        "server": {
+            "type": "wms",
+            "layerType": "dynamic",
+            "url": "http://sampleserver1.arcgisonline.com/ArcGIS/services/Specialty/",
+            "name": "ESRI_StatesCitiesRivers_USA"
+        },
+        "includeLayers": [
+            {
+                "name": "States"
+            },
+            {
+                "name": "Cities"
+            },
+            {
+                "name": "Rivers"
+            }
+        ]
+    },
+    {
+        "displayName": "Global Risk Data Platform",
```

Comment:
I included it because it's a service TNC plans on using. It just started having this error when generating legends a couple hours ago:

`java.lang.OutOfMemoryError: GC overhead limit exceeded GC overhead limit exceeded`